Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/barrel_attachments.yml

```
@@ -26,12 +26,30 @@
     modifiers:
     - conditions:
         wieldedOnly: true
-      accuracyAddMult: 0.2
+      accuracyAddMult: 0.15
```

Extended barrel is supposed to be 0.2 accuracy add mult
```suggestion
      accuracyAddMult: 0.2
```